.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  ruby_rails_test_matrix:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby: ['2.7', '3.0', '3.1']
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        rubygems: 3.3.22
+        bundler-cache: true
+
+    - name: Runs code QA and tests
+      env:
+        RAILS_VERSION: ~> ${{ matrix.rails }}
+        COVERAGE: no
+      run: |
+        sudo apt-get update
+        sudo apt-get install openjdk-8-jdk
+        bundle
+        bundle exec rake rspec
```

```suggestion
      - run: |
          bundle exec rake rspec
        env:
          RAILS_ENV: test
```
can we simplify this to the above? do we need to install `openjdk-8-jdk`?

---

I think technically no, but will need it if bringing the integration suite in (though it may just become a transitive dep when bringing it cassa)

---

I don't think RAILS_ENV matters, `ruby != rails` bro